src/navigate-pop/switching-apps.md

```diff
@@ -0,0 +1,50 @@
+# Navigate Between Windows and Applications
+
+Pop!\_OS uses the GNOME desktop environment, which includes keyboard shortcuts for quickly switching between system and application windows.
+
+---
+
+## Navigate Between Windows
+
+View all windows in the current workspace by pressing `SUPER` + `D`. For more information about Workspaces, see [Using Workspaces](using-workspaces.md)
+
+![Display All Windows](/images/locate-launch-applications/display-all-windows.png)
+
+Use the the `🠐` `🠑` `🠒` `🠓` keys or mouse to move between windows. Select a window with left-click or by pressing `Enter`.
+
+![Select Displayed Window](/images/locate-launch-applications/select-displayed-window.png)
+
+Switch back to the previously selected window by pressing `Alt` + `Esc`.
+
+### Navigate Windows with Keyboard Shortcuts
+
+| Shortcut             | Action |
+|----------------------|--------|
+| <ul><li>SUPER + 🠓</li><li>SUPER + Keypad down</li><li>SUPER + J | Switch focus to the below window. |
+| <ul><li>SUPER + 🠐</li><li>SUPER + Keypad left</li><li>SUPER + H | Switch focus to the left window. |
+| <ul><li>SUPER + 🠒</li><li>SUPER + Keypad right</li><li>SUPER + L | Switch focus to the right window. |
+| <ul><li>SUPER + 🠑</li><li>SUPER + Keypad up</li><li>SUPER + K | Switch focus to the above window. |
+
+
+## Navigate Between Running Applications
+
+Pressing `SUPER` + `Tab` and `Alt` + `Tab` will display a list of running application icons across the screen. Continue holding `SUPER` or `Alt` and pressing `Tab` to toggle between applications. You can also navigate between applications with the `🠐` `🠒`  keys.
```

Comment:
Huh, when I suggested Alt + Tab, I actually thought it was bound to the `Switch windows` action (from searching, it looks like Ubuntu might default it to that.) I didn't realize GNOME on Pop!_OS was now defaulting both Alt + Tab and Super + Tab to the same `Switch applications` action. Apologies.

You could remove the redundant `Alt`s with that info in mind, or if you keep them, I would change all instances of `and` to `or` because one does not need to press _both_ Super + Tab and Alt + Tab for this feature to work, only one or the other.